finance/go-ethereum/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem 1.0
+
+name				go-ethereum
+version				1.6.6
+revision			0
+categories			finance crypto
+license				GPL-3
+platforms			darwin
+maintainers			openmaintainer krischik
+description			Official golang implementation of the Ethereum protocol.
+long_description	Official golang implementation of the Ethereum protocol.
+
+homepage			https://github.com/ethereum/${name}
+master_sites		${homepage}/archive/
+distfiles			v${version}.tar.gz
+distname			go-ethereum-${version}
```

Comment:
The github port group should be used to reduce the complexity of the Portfile. Documentation is in the header comments in the [github port group](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/github-1.0.tcl) or look for other ports using `github.setup`.